File: CSharp/k06_4.cs
using System;

namespace CSharp
{
    public class K06_4 : IExecutable
    {
        public void Execute()
        {
            Console.Write("\t|\t");
            for (int i = 1; i < 10; i++)
            {
                Console.Write(i + "\t");
            }
            Console.WriteLine();

            for (int i = 0; i < 85; i++)
            {
                Console.Write("-");
            }
            Console.WriteLine();

            for (int i = 1; i < 10; i++)
            {
                Console.Write(i + "\t|\t");
                for (int j = 1; j < 10; j++)
                {
                    Console.Write(i * j + "\t");
                }
                Console.WriteLine();
            }
        }
    }
}

File: Python/k06_4.py
class k06_4(object):
    
    def execute(self):
        print('\t|\t', end='')

        for i in range(1, 10):
            print('{}\t'.format(i), end='')
        print('')

        for i in range(1, 85):
            print('-', end='')
        print('')

        for i in range(1, 10):
            print('{}\t|\t'.format(i), end='')
            for j in range(1, 10):
                print('{}\t'.format(i * j), end='')
            print('')
